scala-package/spark/src/test/scala/org/apache/mxnet/spark/MXNetGeneralSuite.scala

```diff
@@ -0,0 +1,50 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.mxnet.spark
+
+import org.apache.spark.SparkContext
+import org.apache.spark.mllib.linalg.Vectors
+import org.apache.spark.mllib.regression.LabeledPoint
+import org.apache.spark.rdd.RDD
+
+class MXNetGeneralSuite extends SharedSparkContext {
+
+  private def parseRawData(sc: SparkContext, path: String): RDD[LabeledPoint] = {
+    val raw = sc.textFile(path)
+    raw.map { s =>
+      val parts = s.split(' ')
+      val label = java.lang.Double.parseDouble(parts(0))
+      val features = Vectors.dense(parts(1).trim().split(',').map(java.lang.Double.parseDouble))
+      LabeledPoint(label, features)
+    }
+  }
+
+  test("run spark with MLP") {
+    val trainData = parseRawData(sc,
+      "/Users/nanzhu/code/mxnet/scala-package/spark/train.txt")
```

Comment:
@szha Any proper places can we put these testing files?

Reply:
any update here about uploading test files?

Reply:
ping for uploading test files @szha ?

Reply:
is there any way I can upload my local copy to somewhere so that you can download it?

Reply:
please make a pull request to dmlc/web-data

Reply:
@szha @piiswrong , thanks ! I have uploaded file at https://github.com/dmlc/web-data/pull/63

Reply:
http://apache-mxnet.s3-accelerate.dualstack.amazonaws.com/gluon/dataset/mxnet-spark-test/train.txt.zip

Reply:
Hi, @szha, would you please help to uncompress and reupload, I zipped it just because github does not allow a large file...I am not sure if jenkins env has unzip command line equipped, etc.

Reply:
@szha ?

Reply:
As requested I decompressed the file and put it here: http://apache-mxnet.s3-accelerate.dualstack.amazonaws.com/gluon/dataset/mxnet-spark-test/train.txt
Why can't you decompress the file in test code?